lighthouse-core/report/html/templates.html

```diff
@@ -325,9 +331,72 @@
         margin-left: 0;
       }
     }
+    .lh-lighthouse__clouds {
+      animation: panacross 30s linear infinite;
+      animation-play-state: paused;
+    }
+    .lh-header-sticky:hover .lh-lighthouse__clouds {
```

Comment:
the `:hover` threw me for a while since the hitbox includes the dropdown of the scores, how much do you like the hover effect vs. always? :)

Reply:
I'm personally ok with always, too :)

Something something lighthouse is always looking out for you metaphor